service/src/main/java/com/ge/predix/acs/policy/evaluation/cache/NoHystrixPolicyEvaluationCacheCircuitBreaker.java

```diff
@@ -0,0 +1,153 @@
+package com.ge.predix.acs.policy.evaluation.cache;
+
+import java.util.List;
+import java.util.Set;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+
+import com.ge.predix.acs.privilege.management.dao.ResourceEntity;
+import com.ge.predix.acs.privilege.management.dao.SubjectEntity;
+import com.ge.predix.acs.rest.PolicyEvaluationResult;
+
+public class NoHystrixPolicyEvaluationCacheCircuitBreaker implements PolicyEvaluationCacheCircuitBreaker {
+    private static final Logger LOGGER = LoggerFactory.getLogger(NoHystrixPolicyEvaluationCacheCircuitBreaker.class);
+
+    @Autowired
+    private PolicyEvaluationCache cacheImpl;
+
+    @Value(value = "${ENABLE_CACHING:false}")
+    private boolean cachingEnabled;
```

Comment:
(same as above  - you could use PolicyEvaluationCacheConfig.isCachingEnabled (add this getter) to avoid setting default value in multiple places.)